### PR TITLE
Switch from OpenID2 to OAuth2

### PR DESCRIPTION
**Pull request by clayton-shopify**
@edward

Google no longer supports OpenID2 authentication, so I've switched this app over to OAuth2.

OAuth2 credentials (which can be obtained from https://console.developers.google.com/) must be placed in the environment variables `OAUTH_CLIENT_ID` and `OAUTH_CLIENT_SECRET`.

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -139,8 +139,8 @@ a { color: #da4c3f; text-transform: uppercase;
   }
 }
 
-#header {
-  margin-top: 100px;
+#header { 
+  @media (max-width: 767px) { margin-top: 100px; }
```
Comment (edward) [inline app/assets/stylesheets/application.css.scss]: Looks like a bad rebase? This looks like it’s reverting a change we wanted recently.

Comment (edward) [inline app/assets/stylesheets/application.css.scss]: Oh – maybe I’m mistaken and this is to turn the login link back on. Right. Nevermind.

**Comment by edward** [comment]
Looks great! Squash commits and merge.

**Comment by clayton-shopify** [comment]
All my commits are squashed now. I left the revert separate so it's clear what happened there.

**Pull request merged**